en/docs/get-started/quick-start-guide.md

```diff
@@ -460,35 +461,34 @@ A message appears to pick a scenario.
     
 3. Note down the `API key` and `secret` for later use.
 
-    ![create-client-id](../assets/img/tutorials/create-client-id.png)
+    ![create-client-id](../../../assets/img/get-started/create-client-id.png)
 
     !!! tip
         In order to avoid getting the following error message, add `localhost.com`  to the authorized domains list.   
         ```
         If Invalid Redirect: domain must be added to the authorized domains list before submitting.
         ```
 
-![authorized domains list](../assets/img/tutorials/authorized-domains-list.png)
+![authorized domains list](../../../assets/img/get-started/authorized-domains-list.png)
```

Comment:
this image is displaying the new console

Reply:
This is a Google console screenshot.